content/en/monitors/monitor_types/process.md

```diff
@@ -11,74 +11,48 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
 ## Overview
 
-Process monitors are configured to check the status of individual or groups of processes. Process monitors are evaluated every minute.
+Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert you based on the behavior of any group of processes across your hosts or tags.
 
-## Live Process Monitors
+## Monitor creation
 
-{{< img src="monitors/monitor_types/process/live_process_monitor_select.png" alt="live process monitor" responsive="true" style="width:35%;">}}
+To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
 
-Live Process Monitors are based on data collected by the [Process Agent][1] which allows you to centrally create monitors that warn or alert based on the behavior of any group of processes across any hosts or tags.
+### Select processes
 
-### Configuration
+Search for a process to monitor using a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure. Use tags to refine your monitor to a specific scope.
```

Comment:
we also support AND/OR/NOT operators in the query string.

https://docs.datadoghq.com/graphing/infrastructure/process/?tab=linuxwindows#search-syntax